elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -602,12 +606,13 @@ export class PeerPool extends EventEmitter {
 
 	private _selectPeersForEviction(): Peer[] {
 		const peers = [...this.getPeers(InboundPeer)].filter(peer => {
-			const found = this._peerLists.whitelisted.find(
+			const found = !!this._peerLists.whitelisted.find(
```

Comment:
This is hard to read. Is there any way to improve readability here?

Reply:
The last one was more readable of course, @shuse2 agrees then I can revert the changes. I'm fine with both.

Reply:
find returns `undefined` if not found, and changing it to `boolean`.
I think it's common expression in JS

Reply:
can also do something like `.filter(peer => this._peerLists.whitelisted.every(p  => constructPeerIdFromPeerInfo(p) !== peer.id)` to make it shorter